src/api/CommentApi.php

```diff
@@ -256,68 +268,68 @@ public function doGetRaw() {
 		);
 	}
 
-	public function doEdit() {
+	public function doEdit(): void {
 		$comment = Comment::newFromId( $this->getInt( 'comment_id' ) );
-		$text = $this->getMain()->getVal( 'text' );
-		if ( $comment ) {
-			$res = CommentBoard::editComment( $comment, $this->getUser(), $text );
-			$this->getResult()->addValue( null, 'result', 'success' );
```

Comment:
I added status check for performed actions, following existing pattern (`result => success / failure`).